### PR TITLE
Task - Improve code quality (Parser/Command)

### PR DESCRIPTION
**Pull request by bgopi23**
* Removed unused import statements and methods
* Added JavaDocs for methods and classes that did not have them
* Reduced clutter by importing message statements so repetition was minimised (e.g. `Messages.MESSAGE_ERROR` -> `MESSAGE_ERROR`)
* Created methods to reduce repetition in various parser classes

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ArgumentMultimap.java [hunk 1]
```diff
@@ -149,21 +154,20 @@ public boolean containsAny(Prefix... prefixes) {
...
     /**
-     * Checks if the preamble of the argumentMultimap object is empty
+     * Checks if the preamble of ArgumentMultimap object is made of 1 part
      *
-     * @return true if the preamble is empty`
+     * @return true if the preamble is by itself
      */
-    public boolean isPreambleEmpty() {
-        return this.getPreamble().isEmpty();
+    public boolean isPreambleAlone() {
```
Comment (dillontkh) [inline src/main/java/seedu/address/logic/parser/ArgumentMultimap.java:170]: The name `isPreambleAlone` gives me the impression that there is only preamble in the command. e.g. `edit 1 2 3 4 5` would return `true` as there is are no prefixes, only the command word and preamble.

Perhaps a more suitable name could be `hasOnlyOnePreambleSegment`?

File: src/main/java/seedu/address/logic/messages/HelpCommandMessages.java [hunk 2]
```diff
@@ -6,9 +6,11 @@
 public class HelpCommandMessages extends Messages {
 
     public static final String COMMAND_WORD = "help";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+    public static final String SHOWING_HELP_MESSAGE = "Launched help window.";
```
[the remark this answers is not in the document]
Comment (bgopi23) [reply src/main/java/seedu/address/logic/messages/HelpCommandMessages.java:9]: good catch!

File: src/main/java/seedu/address/logic/parser/ArgumentMultimap.java (hunk 1 above)
Comment (bgopi23) [reply src/main/java/seedu/address/logic/parser/ArgumentMultimap.java:170]: thanks for the sugestion

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM!

**Review by RyanNgWH — APPROVED** [review]
LGTM!

**Pull request merged**